docs/make_work.jl

```diff
@@ -56,8 +61,20 @@ function setup_experimental_package(Oscar::Module, package_name::String)
   
   # Prepend path
   prefix = "Experimental/" * package_name * "/"
-  result = setup_experimental_prefix(exp_doc, prefix)
-  return result
+  try
+    result = setup_experimental_prefix(Oscar, exp_doc, prefix)
+    return result
+  catch exception
+    msg = sprint(showerror, exception)
+    @warn("Documentation for $package_name errored with the following message:
+>> $msg <<
+Possibly the symlink 
+>> $symlink_src <<
+could not be set, because the directory already exists. Please consider
+removing it. The documentation of $package_name will be empty until this is
+resolved.")
```

Comment:
How about instead something like this in the code which creates the symlink right now:
```julia
  symlink_link = joinpath(Oscar.oscardir, "docs", "src", "Experimental", package_name)
  symlink_target = "../../../experimental/$package_name/docs/src"
  if ispath(symlink_link)
    if !islink(symlink_link) || readlink(symlink_link) != symlink_target
      error("TODO")
    end
  else
    symlink(symlink_target, symlink_link)
  end
```

Reply:
Thanks, I adjusted accordingly. The error message now looks like:
```
ERROR: /home/lars/repos/Oscar.jl/docs/src/Experimental/FTheoryTools already exists, but is not a symlink to ../../../experimental/FTheoryTools/docs/src
Please investigate the contents of /home/lars/repos/Oscar.jl/docs/src/Experimental/FTheoryTools,
optionally move them somewhere else and delete the directiory once you are done.
```
What do you think? A warning would be softer and make sure things succeed, but on the other hand many users would probably be confused why their docs don't show up, so a hard error might be the way to go.